pkg/virt-controller/watch/vm/fuzz/fuzz_test.go

```
@@ -0,0 +1,376 @@
+			}
+			defer namespaceInformer.GetStore().Delete(virtualMachine.Namespace)
+			vms = append(vms, virtualMachine)
+		}
+		// There is no point in continuing
+		// if we have not created any resources.
+		if len(vms) == 0 {
+			return
+		}
+
+		virtClient := kubecli.NewMockKubevirtClient(gomock.NewController(t))
+		virtFakeClient := fake.NewSimpleClientset()
+		// enable /status, this assumes that no other reactor will be prepend.
+		// if you need to prepend reactor it need to not handle the object or use the
+		// modify function
+		virtFakeClient.PrependReactor("update", "virtualmachines",
+			vm.UpdateReactor(vm.SubresourceHandle, virtFakeClient.Tracker(), vm.ModifyStatusOnlyVM))
+		virtFakeClient.PrependReactor("update", "virtualmachines",
+			vm.UpdateReactor(vm.Handle, virtFakeClient.Tracker(), vm.ModifyVM))
+
+		virtFakeClient.PrependReactor("patch", "virtualmachines",
+			vm.PatchReactor(vm.SubresourceHandle, virtFakeClient.Tracker(), vm.ModifyStatusOnlyVM))
+		virtFakeClient.PrependReactor("patch", "virtualmachines",
+			vm.PatchReactor(vm.Handle, virtFakeClient.Tracker(), vm.ModifyVM))
+
+		for _, virtualMachine := range vms {
+			virtClient.EXPECT().VirtualMachineInstance(virtualMachine.Namespace).Return(
+				virtFakeClient.KubevirtV1().VirtualMachineInstances(virtualMachine.Namespace),
+			).AnyTimes()
+		}
+
+		dataVolumeInformer, dvCs := testutils.NewFakeInformerFor(&cdiv1.DataVolume{})
+		dataSourceInformer, dsCs := testutils.NewFakeInformerFor(&cdiv1.DataSource{})
+		vmiInformer, viCs := testutils.NewFakeInformerWithIndexersFor(&v1.VirtualMachineInstance{}, virtcontroller.GetVMIInformerIndexers())
+		vmInformer, vCs := testutils.NewFakeInformerWithIndexersFor(&v1.VirtualMachine{}, virtcontroller.GetVirtualMachineInformerIndexers())
+		pvcInformer, piCs := testutils.NewFakeInformerFor(&k8sv1.PersistentVolumeClaim{})
+
+		for _, virtualMachine := range vms {
+			err := vmInformer.GetIndexer().Add(virtualMachine)
+			if err != nil {
+				return
```

Silently swallowing the err?